File: beginner/__1000__/1005/1005.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;

public class Main {
	public static void main(String[] args) throws IOException {
		InputStreamReader ir = new InputStreamReader(System.in);
		BufferedReader in = new BufferedReader(ir); 

		double a = Double.parseDouble(in.readLine());
		double b = Double.parseDouble(in.readLine());
		double average = ((a * 3.5) + (b * 7.5)) / 11;

		System.out.printf("MEDIA = %.5f\n", average);
	}
}

File: beginner/__1000__/1005/1005.cs
using System;
using System.Globalization;

public class URI 
{
    public static void Main(string[] args)
    {
        double n = double.Parse(Console.ReadLine());
        double m = double.Parse(Console.ReadLine());

        double average = (3.5 * n + 7.5 * m) / 11.0;

        Console.WriteLine("MEDIA = {0}", average.ToString("F5", CultureInfo.InvariantCulture));
    }
}